admin/sql/create_primary_keys.sql

```diff
@@ -2,5 +2,6 @@ BEGIN;
 
 ALTER TABLE "user" ADD CONSTRAINT user_pkey PRIMARY KEY (id);
 ALTER TABLE listen ADD CONSTRAINT listen_pkey PRIMARY KEY (user_id, ts);
+ALTER TABLE sessions ADD CONSTRAINT sessions_pkey PRIMARY KEY (sid);
 
```

Comment:
You have no indexes on the session table. You need to add appropriate indexes for the types of queries you're going to make.